js/interactive-guides/microprofile-config/playground-callback.js

```diff
@@ -0,0 +1,142 @@
+var playground = (function() {
+    
+    var properties = {};
+
+    /**
+     * 
+     * @param {*} key - 
+     * @param {*} value - 
+     * @param {*} source - 
+     * @param {*} ordinal (optional?) - provided ordinal number. otherwise default based on source
+     */
+    var playgroundAddConfig = function(key, value, source, ordinal) {
+        if (!ordinal) {
+            ordinal = __getDefaultOrdinal(source);
+        }
+
+        if (properties[key]) {
+            var oldOrdinal = properties[key]['ordinal'];
+            if (parseInt(ordinal) >= parseInt(oldOrdinal)) {
+                properties[key] = {'ordinal': ordinal, 'value': value};
+            }
+        } else {
+            properties[key] = {'ordinal': ordinal, 'value': value};
+        }
+    };
+
+    var repopulatePlaygroundConfigs = function(tabbedEditor) {
+        properties = {};
```

Comment:
When will this be called? Maybe we could add onChange listeners to the different tabs to update their respective ordinals only.

Reply:
Currently not used it. It should be called upon each update of any of the editor files.